page/new-innovations.php

```diff
@@ -0,0 +1,109 @@
+                                <div class="hide">
+                                    <?php include(str_replace("../../", "", $data[3])) ?>
+                                </div>
+                            </div>
+                        </div>
+                        <div class="col-md-6">
+                            <a href="index.php?page=view-news&id=<?php echo $data[0] ?>"><img src="<?php echo $thumbnail ?>" class="img-fluid" alt=""></a>
+                        </div>
+                    </div>
+
+                    <?php
+                }
+            ?>
+
+            
+        </div>
+
+        <div class="col-md-4">
+            <div class="pb-5">
+                <h5><dt><b>RUANG PUBLIK</b></dt></h5>
+                <ul class="fa-ul list-aside my-3">
+                    <li><span class="fa-li"><i class="fas fa-angle-right"></i></span><a href="index.php?page=news">BERITA</a></li>
+                    <li><span class="fa-li"><i class="fas fa-angle-right"></i></span><a href="index.php?page=press-release">SIARAN PERS</a></li>
+                    <li><span class="fa-li"><i class="fas fa-angle-right"></i></span><a href="index.php?page=pindad-in-news">PINDAD DALAM BERITA</a></li>
+                    <li><span class="fa-li"><i class="fas fa-angle-right"></i></span><a href="index.php?page=majalah">MAJALAH</a></li>
+                    <li><span class="fa-li"><i class="fas fa-angle-right"></i></span><a href="index.php?page=catalog-and-brochures">KATALOG DAN BROSUR</a></li>
+                    <li><span class="fa-li"><i class="fas fa-angle-right"></i></span><a href="index.php?page=gallery">GALLERY</a></li>
+                    <li><span class="fa-li"><i class="fas fa-angle-right"></i></span><a href="index.php?page=video">VIDEO</a></li>
+                </ul>
+            </div>
+
+            <div class="pb-5">
+                <h5><dt><b>POPULAR POST</b></dt></h5>
+
+                <?php
+
+                $popular_post = mysqli_query($connection, "select * from berita order by tanggal desc");
+                $i = 0;
+                while($data = mysqli_fetch_array($popular_post)){
+                    if($i == 4){
+                        break;
+                    }
+                    if($data[4]==null || $data[4]==""){
+                        $thumbnail = "#";
+                    }else{
+                        $thumbnail = "news/".explode("-", $data[4])[0]."/gambar/".explode("-", $data[4])[1];
+                    }
+                    ?>
+
+                    <div class="row popular-post my-4">
+                        <div class="col-md-3">
+                            <a href="index.php?page=view-news&id=<?php echo $data[0] ?>"><img src="<?php echo $thumbnail ?>" alt=""></a>
+                        </div>
+                        <div class="col-md-9">
+                            <div class="text-warning font-weight-bold tanggal"><?php echo date("d M, Y", strtotime($data[2])) ?></div>
```

Comment:
Codacy found an issue: [Use of echo language construct is discouraged.](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5762293)